### DISCUSSION TITLE
Does docker image include Darktable?

### DISCUSSION THREAD

**Discussion by Europa2010AD** [photoview/photoview#580] [Q&A and Troubleshooting]
I've installed Photoview via docker-compose, however my .NEF raw files don't seem to have been processed by Photoview. Is Darktable part of the docker image? If not, how do I go about enabling it via docker-compose?

**Comment by viktorstrate** [comment]
Darktable is included for AMD CPUs but not ARM (eg. RaspberryPi). This is because Darktable does not run on that platform

**Reply by Europa2010AD** [reply]
I'm running the docker-compose on my NAS (QNAP TVS-951X) which has an Intel Celeron 3865U CPU, so this should mean Darktable is already included, right?

**Reply by viktorstrate** [reply]
Yes, then it should be included and `.nef` files are supported.

To troubleshoot you can look at the logs by running `docker-compose logs -f` and then start the scanner and see if it reports anything. When photoview starts, it also logs whether or not Darktable was enabled.
